src/index.ts

```diff
@@ -0,0 +1,16 @@
+// import { getAllTransactionsWithin } from './queries';
+
+import { getWaleltsEligibleForStreak } from './queries';
+
+// const minBlock = 916705;
+// const maxBlock = 916756;
+
+if (require.main === module) {
```

Comment:
why do we need that?
can you also clean the comments?

Reply:
There will be our main function call (unless we want this package to be a module) where we will read the argv and decide which method to run, maybe :)